Makefile

```diff
@@ -1,6 +1,31 @@
+current_dir = $(shell pwd)
+
+.PHONY: check
 check:
 	flake8 --config .flake8-code .
 	flake8 --config .flake8-doc .
 	pylint lookout
 
-.PHONY: check
+.PHONY: docker-build
+docker-build: Dockerfile
+	docker build -t srcd/style-analyzer .
+
+.PHONY: docker-test
+docker-test:
+	$(MAKE) _docker-test || docker rm -f style_analyzer_bblfshd
+
+.PHONY: _docker-test
+_docker-test:
+	docker run -d --rm --name style_analyzer_bblfshd \
+		--privileged -p 9432\:9432 bblfsh/bblfshd\:v2.5.0
+	docker exec style_analyzer_bblfshd \
+		bblfshctl driver install javascript docker://bblfsh/javascript-driver\:v1.2.0
+	docker run --rm -it --network host \
```

Comment:
We should build an image with tests included rather than mount the directories inside. Much stuff is hardcoded, will be complex to maintain. So:

```make
docker-test: docker-build
    docker ps | grep bblfshd  # do not launch bblfshd each time, require it
    docker run --rm -it --network host --entrypoint python3 -w /style-analyzer srcd/style-analyser -m unittest discover
```